packages/gatsby-theme-api-docs/src/components/api/resource/method/url-scopes-responses/url-scopes-responses.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
+import { customProperties } from '@commercetools-uikit/design-system';
 
 import { responseRepresentation } from '../../../../../utils/constants';
 import Scopes from './scopes';
 import Responses from './responses';
 
 const Container = styled.div`
-  padding: 1rem;
+  padding: ${customProperties.spacingM};
```

Comment:
We should use here the spacing from the docs uikit. 

```
designSystem.dimensions.spacings.m
```

Reply:
I see, thanks.